client/src/components/RelatedProducts/Related-Items.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import $ from 'jquery';
+import ProductCard from './Product-Card.jsx';
+import { Carousel } from '@trendyol-js/react-carousel';
+import { Item } from  './Product-Card.jsx';
+
+class RelatedItems extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      currentItem : 90,
+      relatedItems : []
+    };
+  }
+
+  componentDidMount() {
+    $.ajax({
+      url: '/relatedItems',
+      method: 'GET',
+      data: {item_id: this.state.currentItem},
+      success: data => {
+        console.log('This is coming back to the front end: ', data)
```

Comment:
Sorry I forgot this one in my initial review, please comment the console log out so we can keep everyone's terminals clear.